File: Golang/ContainerWithMostWater/ContainerWithMostWater_test.go
package ContainerWithMostWater

import "testing"

func TestMaxArea(t *testing.T) {

	if maxArea([]int{1, 1}) == 1 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if maxArea([]int{1, 4, 9}) == 4 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if maxArea([]int{1, 4, 9, 4}) == 8 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/ContainerWithMostWaterTest.py
from unittest import TestCase

from ContainerWithMostWater import ContainerWithMostWater


class TestContainerWithMostWater(TestCase):
    def test_maxArea(self):
        cwmw = ContainerWithMostWater()

        self.assertTrue(cwmw.maxArea([1, 1]) == 1)

        self.assertTrue(cwmw.maxArea([1, 4, 9]) == 4)

        self.assertTrue(cwmw.maxArea([1, 4, 9, 4]) == 8)
